README

```diff
@@ -113,6 +113,13 @@ m_addr_email
        (http://red.roses.de/~clemens/addressbook/) Tk program to
        search for addresses.
 
+m_mu
+       This module uses the program mu
+       (http://www.linuxstart.com/~jheinonen/abook/), a tool for
```

Comment:
This URL redirects to a non existing host at the moment. Maybe http://www.djcbsoftware.nl/code/mu/ is a better URL here?

Reply:
Whoops. Yes, you're right.